### PR TITLE
(DOCSP-28697): Add docs release notes for w/e March 24

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-28697

### CODE REVIEW

File: docs-release-notes.md [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+- Sync/Set the Client Log Level: New page with tested, Bluehawked code examples showing how to set the log level and how to set a custom logger
+- Update the Kotlin SDK version to 1.7.0
+- Add Partition-Based Sync code examples for configuring and opening a realm that were erroneously deleted
+
+## .NET SDK
+
+- Quick Start: Add a line about quick start demonstrating adding Device Sync
+- Quick Start for Unity: Rename "Integrate Realm with Unity" to "Quick Start for Unity", move it higher in the Table of Contents, move a note about multiprocess-sync-related crashes
+- Realm Files/Reduce Realm File Size: Add information about automatic compaction, clarify details for manual compaction
+- Manage Users/Manage Email/Password Users: Update details around password reset, particularly around calling a password reset function
```
Comment (cbullinger) [inline docs-release-notes.md:28]:
```suggestion
- Manage Users/Manage Email/Password Users: Update details around calling a password reset function
```

File: docs-release-notes.md [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+- Use Realm in a Console App: Add details around Device Sync, update code example
+- Compatibility: Add `tvOS` to the list of supported platforms
+
+## Node.js SDK
+
+- Usage Examples/Manage Email/Password Users: Update details around password reset, particularly around calling a password reset function
```
Comment (cbullinger) [inline docs-release-notes.md:34]:
```suggestion
- Usage Examples/Manage Email/Password Users: Update details around calling a password reset function
```

File: docs-release-notes.md [hunk 3]
```diff
@@ -1,3 +1,60 @@
...
+
+## React Native SDK
+
+- Major rewrite across the SDK to use `@realm/react` in all code examples and accompanying text
+- Manage Users/Manage Email/Password Users: Update details around password reset, particularly around calling a password reset function
```
Comment (cbullinger) [inline docs-release-notes.md:39]:
```suggestion
- Manage Users/Manage Email/Password Users: Update details around calling a password reset function
```

File: docs-release-notes.md [hunk 4]
```diff
@@ -1,3 +1,60 @@
...
+
+## Swift SDK
+
+- Realm Files/Reduce Realm File Size: Add information about automatic compaction, clarify details for manual compaction
+- Manage Users/Manage Email/Password Users: Update details around password reset, particularly around calling a password reset function
```
Comment (cbullinger) [inline docs-release-notes.md:44]:
```suggestion
- Manage Users/Manage Email/Password Users: Update details around calling a password reset function
```

**Review by cbullinger — APPROVED** [review]
one small rewording suggestion

File: docs-release-notes.md [hunk 5]
```diff
@@ -1,3 +1,60 @@
...
+## C++ SDK
+
+- Manage Users/Manage Email/Password Users: Minor tweaks based on what's currently supported
+
+## Flutter SDK
+
+- Realm Database/Manage Realm Database Files/Reduce Realm File Size
+  - Note that you can compact either a non-synced or synced realm
+  - Add information about automatic compaction, clarify details for manual compaction
+- Manage Users/Email/Password Users: Update details around password reset, particularly around calling a password reset function
```
[the remark this answers is not in the document]
Comment (dacharyc) [reply docs-release-notes.md:12]: Good note! The changes are across the password reset section, though, not only in the password reset function part of things, so this change isn't entirely accurate.

**Pull request merged**